_posts/2021-12-15-WildFly26-Final-Released.adoc

```diff
@@ -0,0 +1,128 @@
+
+To override the value of a simple attribute of a management resource, you can specify an environment variable with the following conversion mapping:
+
+1. Take the address of the resource.
+2. Remove the leading slash (`/`).
+3. Append *two underscores* (`__`) and the name of the attribute.
+4. Replace all non-alphanumeric characters with an underscore (`_`) 
+5. Transform to upper case.
+
+For example, to set the value of the `proxy-address-forwarding` attribute to `true` on the `/subsystem=undertow/server=default-server/http-listener=default` resource, you can use the following environment variable:
+
+[source]
+----
+export SUBSYSTEM_UNDERTOW_SERVER_DEFAULT_SERVER_HTTP_LISTENER_DEFAULT__PROXY_ADDRESS_FORWARDING=true
+----
+
+This feature is not activated by default. To activate this feature, you must set the `WILDFLY_OVERRIDING_ENV_VARS` environment variable on your target platform:
+
+[source]
+----
+export WILDFLY_OVERRIDING_ENV_VARS=1
+----
+
+== Other New Features
+
+* The Elytron subsystem has added a new https://github.com/wildfly/wildfly-proposals/blob/main/elytron/WFCORE-5483-custom-loginmodule-compatible-sec-realm.adoc[JAAS security realm type]. The JAAS security realm can be used to configure custom Login Modules in the Elytron subsystem to use for credential verification.
+This is done by initializing and using a LoginContext from a provided JAAS configuration file to verify user's credentials.
+* A WildFly domain mode `Host Controller` can now https://github.com/wildfly/wildfly-proposals/blob/main/core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc[resolve standard server system properties] when launching a managed server JVM. This makes it easier to configure shared server JVM settings at the domain, server-group or host level, while still having those settings incorporate server-specific path values. This is useful for things like adding GC logging to the server launch command, where the setting needs to include a server-specific location to write the log file.
+* The link:https://github.com/wildfly/quickstart/tree/26.0.0.Final/microprofile-reactive-messaging-kafka[MicroProfile Reactive Messaging quickstart] has been enhanced to link:https://github.com/wildfly/wildfly-proposals/blob/main/microprofile/WFLY-14800-MicroProfile_Reactive_Messaging_QuickStart_Update_For_2.0.adoc[demonstrate new capabilities that came with our integration of MicroProfile Reactive Messaging 2.0].
+
+== Compatibility Notes
+
+There are a couple of compatibility topics that I want to highlight:
+
+* To build a link:https://docs.wildfly.org/bootablejar/[Bootable JAR] with WildFly 26, you will need to upgrade the link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin[Bootable JAR Maven plugin] to the 6.1.1.Final or later release, preferably to link:https://repository.jboss.org/org/wildfly/plugins/wildfly-jar-maven-plugin/7.0.0.Final/[7.0.0.Final].
```

Comment:
@bstansberry , the root cause of this upgrade is caused by https://github.com/wildfly-extras/wildfly-jar-maven-plugin/issues/285